blog/exporters.md

```diff
@@ -74,48 +78,102 @@ The `secadvisor.yml` is used as a parameter when running secadvisor from the com
 The default `secadvisor.yml` has the following fields:
 
 ```
+# host_id is used to reference the agent, should be unique or empty
 host_id: ""
+# list of analyzers for running REST requests
 analyzers:
-  127.0.0.1:8082
+  - 127.0.0.1:8082
 pipeline:
+  # analyzer credentials for websocket connection
   analyzer:
     subscriber_url: ws://127.0.0.1:8082/ws/subscriber/flow
     subscriber_username:
     subscriber_password:
+  # classify flows into: internal, ingress, egress and other
   classify:
-    # cluster_net_masks:
-      # - 10.0.0.0/8
-      # - 172.16.0.0/12
-      # - 192.168.0.0/16
+    # list of internal cluster address ranges
+    cluster_net_masks:
+      - 10.0.0.0/8
+      - 172.16.0.0/12
+      - 192.168.0.0/16
+  # filter out flows which match a criteria
   filter:
-    # excluded_tags:
-      # - internal
-      # - other
-      # - ingress
-      # - egress
+    # exclude the following classified tags 
+    excluded_tags:
+      - other
+  # transform to sa record structure
   transform:
+    type: sa
     sa:
-      # exclude_started_flows: true
+      exclude_started_flows: true
+  encode:
+    type: json
+    json:
+      pretty: true
+  compress:
+    type: gzip
   store:
     type: s3
     s3:
       # -- client parames --
       endpoint: http://127.0.0.1:9000
-      region: local
-      bucket: bucket
       access_key: user
       secret_key: password
+      region: local
+      bucket: bucket
       # api_key: key
       # iam_endpoint: https://iam.cloud.ibm.com/identity/token
       object_prefix: logs
+
       # -- bulk store params --
       max_flows_per_object: 6000
       max_seconds_per_object: 60
       max_seconds_per_stream: 86400
       max_flow_array_size: 100000
 ```
 
-### Description of fields in `secadvisor.yml` file
+### Description of fields in `exporter.yml` file
+
+#### Synopsis
+
+| Field Name            | Type           | Description |
+|-----------------------|----------------|-------------|
+| host_id               | string         | "" or unique identifier |
+| analyzers             | URL            | address of Skydive analyzer for queries |
+| pipeline              | list of params | pipeline specification |
+| analyzer              | list of params | analyzer credentials for websocket connection |
+| subscriber_url        | URL            | websocket address to obtain flows from Skydive |
+| subscriber_username   | string         | websocket creedentials |
+| subscriber_password   | string         | websocket creedentials |
+| cluster_net_masks     | list of cidr   | list of net-addr/net_mask to be considered internal addresses |
+| filter                | list of params | filter options; currently the only parameter is `excluded_tags` |
```

Comment:
redundant stating that filter has excluded_tags here when a line below you have an entry for excluded_tags

Reply:
At present it is redundant because we have only one option. If we don't expect any future changes, then these could be combined into a single entry. The fact that we have them separated enables the possibility of adding future suboptions.

Reply:
I expect more options in future, but no need that at the top-level `filter` description to repeat the suboptions.